2021-08-29 10:05:18,020 INFO    MainThread:5498 [wandb_setup.py:_flush():69] setting env: {}
2021-08-29 10:05:18,020 INFO    MainThread:5498 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-29 10:05:18,021 INFO    MainThread:5498 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/code/EfficientNet_cross_level_1/wandb/run-20210829_100518-1wt7ntmg/logs/debug.log
2021-08-29 10:05:18,021 INFO    MainThread:5498 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/code/EfficientNet_cross_level_1/wandb/run-20210829_100518-1wt7ntmg/logs/debug-internal.log
2021-08-29 10:05:18,021 INFO    MainThread:5498 [wandb_init.py:init():375] calling init triggers
2021-08-29 10:05:18,021 INFO    MainThread:5498 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'epochs': 4, 'batch_size': 128, 'learning_rate': 0.05}
2021-08-29 10:05:18,021 INFO    MainThread:5498 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 3geahu2g
2021-08-29 10:05:18,023 INFO    MainThread:5498 [wandb_run.py:finish():1271] finishing run simbean/123/3geahu2g
2021-08-29 10:05:18,057 INFO    MainThread:5498 [jupyter.py:save_history():353] not saving jupyter history
2021-08-29 10:05:18,057 INFO    MainThread:5498 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-08-29 10:05:18,057 INFO    MainThread:5498 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-08-29 10:05:18,058 INFO    MainThread:5498 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-08-29 10:05:18,058 INFO    MainThread:5498 [wandb_run.py:_restore():1543] restore
2021-08-29 10:05:18,393 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 947
}

2021-08-29 10:05:18,639 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 947
}

2021-08-29 10:05:18,895 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 4259
}

2021-08-29 10:05:18,998 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10344
}

2021-08-29 10:05:19,104 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10344
}

2021-08-29 10:05:19,207 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10344
}

2021-08-29 10:05:19,309 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10344
  total_bytes: 10344
}

2021-08-29 10:05:19,412 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10344
  total_bytes: 10344
}

2021-08-29 10:05:19,515 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10344
  total_bytes: 10344
}

2021-08-29 10:05:19,617 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10344
  total_bytes: 10344
}

2021-08-29 10:05:19,718 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10344
  total_bytes: 10344
}

2021-08-29 10:05:20,107 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10344
  total_bytes: 10344
}

2021-08-29 10:05:20,209 INFO    MainThread:5498 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10344
  total_bytes: 10344
}

2021-08-29 10:05:21,323 INFO    MainThread:5498 [wandb_run.py:_show_files():1915] logging synced files
2021-08-29 10:05:21,328 INFO    MainThread:5498 [wandb_init.py:init():424] starting backend
2021-08-29 10:05:21,328 INFO    MainThread:5498 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-29 10:05:21,331 INFO    MainThread:5498 [backend.py:ensure_launched():135] starting backend process...
2021-08-29 10:05:21,360 INFO    MainThread:5498 [backend.py:ensure_launched():139] started backend process with pid: 8379
2021-08-29 10:05:21,363 INFO    MainThread:5498 [wandb_init.py:init():429] backend started and connected
2021-08-29 10:05:21,380 INFO    MainThread:5498 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-08-29 10:05:21,380 INFO    MainThread:5498 [wandb_run.py:_label_probe_notebook():822] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-08-29 10:05:21,381 INFO    MainThread:5498 [wandb_init.py:init():477] updated telemetry
2021-08-29 10:05:21,382 INFO    MainThread:5498 [wandb_init.py:init():500] communicating current version
2021-08-29 10:05:22,144 INFO    MainThread:5498 [wandb_init.py:init():505] got version response 
2021-08-29 10:05:22,144 INFO    MainThread:5498 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-29 10:05:22,357 INFO    MainThread:5498 [wandb_init.py:init():540] starting run threads in backend
2021-08-29 10:05:26,859 INFO    MainThread:5498 [wandb_run.py:_console_start():1601] atexit reg
2021-08-29 10:05:26,859 INFO    MainThread:5498 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-08-29 10:05:26,860 INFO    MainThread:5498 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-08-29 10:05:26,861 INFO    MainThread:5498 [wandb_run.py:_redirect():1536] Redirects installed.
2021-08-29 10:05:26,861 INFO    MainThread:5498 [wandb_init.py:init():565] run started, returning control to user process
2021-08-29 10:05:32,767 INFO    MainThread:5498 [wandb_setup.py:_flush():69] setting env: {}
2021-08-29 10:05:32,768 INFO    MainThread:5498 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-29 10:05:32,768 INFO    MainThread:5498 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/code/EfficientNet_cross_level_1/wandb/run-20210829_100532-3egjct86/logs/debug.log
2021-08-29 10:05:32,768 INFO    MainThread:5498 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/code/EfficientNet_cross_level_1/wandb/run-20210829_100532-3egjct86/logs/debug-internal.log
2021-08-29 10:05:32,769 INFO    MainThread:5498 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f48f8dab220>
2021-08-29 10:05:32,769 INFO    MainThread:5498 [wandb_init.py:init():375] calling init triggers
2021-08-29 10:05:32,769 INFO    MainThread:5498 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'epochs': 4, 'batch_size': 128, 'learning_rate': 0.05}
2021-08-29 10:05:32,769 INFO    MainThread:5498 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 1wt7ntmg
2021-08-29 10:05:32,770 INFO    MainThread:5498 [wandb_run.py:finish():1271] finishing run simbean/123/1wt7ntmg
